src/components/UI/AnnoucementBar/AnnoucementBar.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { AnnoucementBar } from "./AnnoucementBar";
+import { ReactRouterLinkButtonSolidOrangeWhite } from "./../../UI/Button";
+import { Provider } from "react-redux";
+import initStore from "./../../../store";
+import { ConnectedRouter } from "connected-react-router";
+import { Route, Switch } from "react-router-dom";
+import { createBrowserHistory } from "history";
+
+const history = createBrowserHistory();
+const store = initStore(history);
+
+export default {
+  title: "UI/AnnoucementBar",
+  component: AnnoucementBar,
+  parameters: {
+    componentSubtitle: "Banner for any announcements.",
+  },
+};
+
+export const Default = () => {
+  return (
+    <Provider store={store}>
+      <ConnectedRouter history={history}>
+        <Switch>
+          <Route path="/">
+            <AnnoucementBar className="bg-brand-navy text-white">
+              <div className="row align-items-center">
+                <div className="col-12">
+                  <p className="mb-0">Empty banner</p>
+                </div>
+              </div>
+            </AnnoucementBar>
+          </Route>
+        </Switch>
+      </ConnectedRouter>
+    </Provider>
+  );
+};
+
+export const WebinarSeries = () => {
+  return (
+    <Provider store={store}>
+      <ConnectedRouter history={history}>
+        <Switch>
+          <Route path="/">
+            <AnnoucementBar className="bg-brand-navy text-white" backgroundImage="/static/images/webinar/banner.jpg">
```

Comment:
are we still going to use this webinar announcement?

Reply:
not that i know of for now, was thinking leaving it as a showcase (next time can copy paste?) im not sure : /